global.json

```diff
@@ -1,9 +1,10 @@
 {
   "tools": {
-    "dotnet": "7.0.203"
+    "dotnet": "9.0.100-preview.3.24204.13"
```

Comment:
Is 5his correct? main is now net9?

Reply:
Yeah this is just tools, not dotnet for building maui, same as we use xharness 9.0.0 on main also